test/common/access_log/access_log_impl_test.cc

```diff
@@ -918,6 +923,196 @@ name: envoy.file_access_log
       "response_flag_filter {\n  flags: \"UnsupportedFlag\"\n}\n");
 }
 
+TEST_F(AccessLogImplTest, GrpcStatusFilterValues) {
+  const std::string yaml_prefix = R"EOF(
+name: envoy.file_access_log
+filter:
+  grpc_status_filter:
+    statuses:
+      - )EOF";
+
+  const std::string yaml_suffix = R"EOF(
+config:
+  path: /dev/null
+  )EOF";
+
+  const auto desc = envoy::config::filter::accesslog::v2::GrpcStatusFilter_Status_descriptor();
+  const int grpcStatuses = static_cast<int>(Grpc::Status::GrpcStatus::MaximumValid) + 1;
+  if (desc->value_count() != grpcStatuses) {
+    FAIL() << "Mismatch in number of gRPC statuses, GrpcStatus has " << grpcStatuses
+           << ", GrpcStatusFilter_Status has " << desc->value_count() << ".";
+  }
+
+  std::ostringstream out;
+  for (int i = 0; i < desc->value_count(); i++) {
+    out << yaml_prefix << desc->value(i)->name() << yaml_suffix;
```

Comment:
I think you can just do absl::StrCat or fmt::format, no need of ostringstream.

Reply:
Wasn't aware this existed in the codebase, thanks!

Reply:
Used `fmt::format`, changed in 3fb87e8.